### PR TITLE
Add missing plugin dependency.

### PR DESCRIPTION
**Pull request by lvotypko**
Fix constant failure of test FindBugsPluginTest.should_set_pipeline_result

### CODE REVIEW

File: src/test/java/plugins/FindBugsPluginTest.java [hunk 1]
```diff
@@ -177,6 +177,7 @@ public void should_set_result_to_unstable_if_warning_found() {
      * Builds a pipeline project and checks if it is unstable.
      */
     @Test
+    @WithPlugins("workflow-aggregator")
```
Comment (uhafner) [inline src/test/java/plugins/FindBugsPluginTest.java]: Is it still ok to use workflow-aggregator? Didn't @jglick suggest to not use it anymore and make the dependencies explicit? Or is this guideline just for plug-ins and not ATH?

Comment (lvotypko) [reply src/test/java/plugins/FindBugsPluginTest.java]: Hi, yeah you are right - I totally forgot! I will change it for something with less dependencies.

**Comment by lvotypko** [comment]
Finally all plugins added. I will rebase it in one commit.

**Comment by uhafner** [comment]
Thanks for finding out the required dependencies. This really is quite cumbersome to find the required workflow libs. (I have the same problem in some integration tests...)

**Pull request merged**